db/migrate/20201102163702_add_state_to_external_pull_request.rb

```diff
@@ -0,0 +1,9 @@
+class AddStateToExternalPullRequest < ActiveRecord::Migration[6.0]
+  def change
+    execute <<-SQL
+      CREATE TYPE state AS ENUM ('open', 'closed', 'merged');
+    SQL
```

Comment:
Be careful, because this SQL command is not reversible. You can check out this migration to see how to make it reversible `db/migrate/20200305150445_change_review_requests_status.rb`